packages/moonstone/Scrollable/ScrollableNative.js

```diff
@@ -560,6 +570,60 @@ class ScrollableBaseNative extends Component {
 		}
 	}
 
+	updateFocusAfterVoiceControl = () => {
+		const
+			{direction} = this.props,
+			viewportBounds = this.uiRef.containerRef.getBoundingClientRect(),
+			spotItemBounds = document.activeElement.getBoundingClientRect(),
```

Comment:
Is this safe to refer `activeElement` here? If this function is called when the focus is out of scrollable area, it may lead abnormal behaviors in some cases.

Reply:
Your comment is right. So, I did change to use *Spotlight.getCurrent()* and add to checking whether container contains spot item before comparing position.